src/rdkb/impl.c

```diff
@@ -347,6 +347,7 @@ int Get_Webconfig_URL( char *pString)
 				cpeabStrncpy(pString, tempUrl, strlen(tempUrl)+1);
 			}
 			CpeabsDebug("Get_Webconfig_URL. pString %s\n", pString);
+			CPEABS_FREE(tempUrl);
```

Comment:
Move this free inside tempUrl NULL check after cpeabStrncpy

Reply:
done